src/proofs/ProofSteps.cpp

```diff
@@ -869,7 +583,7 @@ StepHandler::getInstPairs(int stepIndex, vec<Normalizer::Equality> const & stepN
     return res;
 }
 
-int StepHandler::stepReusage(std::shared_ptr<Term> term) {
+int StepHandler::stepReusage(const std::shared_ptr<Term> & term) {
```

Comment:
Why is `stepReusage` returning an `int`, but the callers of this method never do anything with this return value?
Can it be changed to `void`?